.github/workflows/python.yml

```diff
@@ -1,27 +1,30 @@
 name: Python Package
 
 on:
-  pull_request:
   push:
     branches:
-      - '*'
+     - main
     tags:
       - '*'
 
+  pull_request:
+    branches:
+      - main
+
 jobs:
   build:
     runs-on: ubuntu-latest
     name: Build YAMS-Lib and Python Wheel
 
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
```

Comment:
FYI, these were taken from OPR so you may want to update them there too

Reply:
Yes it's why I asked :p